src/commands/run-build.yml

```diff
@@ -10,6 +10,13 @@ parameters:
       in the plan returned by buildfile.m as well as all the tasks on which they depend.
     type: string
     default: ""
+  build-options:
+    description: >
+      MATLAB build options, specified as a list of options separated by spaces. The command
```

Comment:
"specified as a list of space-separated options"?

Reply:
For [GitHub Actions](https://github.com/matlab-actions/run-build?tab=readme-ov-file#run-matlab-build), @mw-hrastega got the `build-options` description approved by the editor and hence I used the same one with minor modifications. Also, just as an fyi, a similar content review is in works for CircleCI.